hera_corr_cm/hera_corr_cm.py

```diff
@@ -197,12 +197,12 @@ def get_eq_coeffs(self, ant, pol):
             return False
         try:
             t = float(v['time'])
-        except:
+        except:  # noqa
```

Comment:
bare excepts aren't great. Can we fix it instead of ignoring the warnings about it?

Reply:
I'll make an issue and take out the noqas